test/integration/unknown.cc

```diff
@@ -83,3 +83,51 @@ TEST(Unknown, CopyUnknownElement)
   EXPECT_EQ("my nested xml", nestedElem->Get<std::string>("attr"));
   EXPECT_EQ("AString", nestedElem->Get<std::string>("string"));
 }
+
+/////////////////////////////////////////////////
+/// Test that elements that aren't part of the spec are flagged with when
+/// UnrecognizedElementsPolicy is set to err
+TEST(UnrecognizedElements, UnrecognizedElementsWithWarningsPolicies)
+{
+  const std::string testFile =
+    sdf::filesystem::append(PROJECT_SOURCE_PATH, "test", "sdf",
+        "unrecognized_elements.sdf");
+
+  sdf::ParserConfig config;
+
+  {
+    config.SetUnrecognizedElementsPolicy(sdf::EnforcementPolicy::ERR);
+    sdf::Root root;
+    const auto errors = root.Load(testFile, config);
+    EXPECT_FALSE(errors.empty());
```

Comment:
Can you add an expectation on at least one of the output messages? Just so it doesn't pass due to other non-related error messages in the future.

Reply:
Done